PhysicsTools/PatAlgos/plugins/PATMuonProducer.cc

```diff
@@ -179,6 +185,10 @@ PATMuonProducer::PATMuonProducer(const edm::ParameterSet & iConfig, PATMuonHeavy
   // MC info
   simInfo_        = consumes<edm::ValueMap<reco::MuonSimInfo> >(iConfig.getParameter<edm::InputTag>("muonSimInfo"));
 
+  triggerObjects_ = consumes<std::vector<pat::TriggerObjectStandAlone>>(edm::InputTag("slimmedPatTrigger"));
+  if (iConfig.exists("hltCollectionNames"))
```

Comment:
please add a default values to ::fillDescriptions.
This will also drop the need to use ::exists, which is strongly discouraged